README.md

```diff
@@ -98,29 +94,69 @@ flags.
 Example:
 
 ```
-make run_debian ... \
-     EXTRA_QEMU_ARGS="-device virtio-net-pci,iommu_platform=on,disable-legacy=on"
+   EXTRA_QEMU_ARGS="-device virtio-net-pci,iommu_platform=on,disable-legacy=on" \
+   ... \
+   scripts/run_debian.sh
 ```
 
 ### Test VM
 
 A pair of test VMs are located in `test-workloads`.
 
-`tellus` is a target build with `make tellus` that runs in VS mode
-and provides the ability to send test API calls to `salus` running in HS mode.
+`tellus` is a target build with `bazel build //test-workloads:tellus_guestvm_rule`
+that runs in VS mode and provides the ability to send test API calls 
+to `salus` running in HS mode.
 
 `guestvm` is a test confidential guest. It is started by `tellus` and used for
-testing the guest side of the TSM API.
+testing the guest side of the TSM API. 
 
-A makefile shortcut is provided:
+Once it has been build, you can use the command below to run it.
 
 ```
-make run_tellus \
-    QEMU=<path-to-qemu-tree>
+    QEMU=<path-to-qemu-build/riscv64-softmmu> \
+    scripts/run_tellus.sh
+```
+
+This will boot salus, tellus, and the guestvm the specified qemu.
+
+QEMU, Linux kernel, Buildroot, and Debian:
+- The same as in Make/Cargo build
+
+### General
+
+
+What were make targets in the Make/Cargo build are now shell scripts.
+
+From the top level directory, run
+```bash
+scripts/run_tellus.sh
+```
+
+Many of the variable can be overwritten using environment variables on the
+command line. For example, to use a different version of qemu and 3 cores,
+you can do the following:
+```bash
+QEMU_PATH=/scratch/qemu-salus2/build/riscv64-softmmu/ NCPU=3 scripts/run_tellus.sh
+```
+
+All the other make targets to run salus with linux work analogously. 
```

Comment:
This block can be omitted as it appears to be a duplicate of what's already stated above.

Reply:
fixed